crates/holochain/src/core/workflow/app_validation_workflow.rs

```diff
@@ -253,13 +254,15 @@ async fn app_validation_workflow_inner(
     // Validate ops sequentially
     for sorted_dht_op in sorted_dht_ops.into_iter() {
         let (dht_op, dht_op_hash) = sorted_dht_op.into_inner();
+        let deps = dht_op.sys_validation_dependencies();
+
         let chain_op = match dht_op {
             DhtOp::ChainOp(chain_op) => chain_op,
-            _ => continue,
+            _ => unreachable!("warrant ops are never sent to app validation"),
```

Comment:
The only reason this would panic is if we the devs accidentally allow this code path to be reached, so I do think this should panic. We should never allow this to happen, and it will certainly make our tests fail if it ever does, and that's good.